### ISSUE TITLE
[SEARCH] Issue when using the search bar

### ISSUE THREAD

**Issue by jfavlam** [MediaComem/museumXTD#33]
On https://www.museumxtd.ch/Navigation/Search/ when searching for 'stratégie' as an example.
It emit the JS error:
```
Uncaught TypeError: Cannot read properties of null (reading 'search')
    at search (search.js:33:27)
    at HTMLInputElement.<anonymous> ((index):70:7)
```
from `search.js`

**Comment by jamest1903** [comment]
Indeed i see the same now also after i pulled the latest changes from the repo, I will investigate.

**Comment by jamest1903** [comment]
Ok so the problem is here in file **Tableau Acteurs et Projets**
...
/!\ Ne pas modifier la première ligne
...

the character \ is read as a special escaping character in the JSON search content. If you wish to keep this content i can see if I can add a check to ignore this special character.

**Comment by jamest1903** [comment]
Ok I have added a check when we create the search JSON content that any escape characters are now removed. I pushed the changes and its working now.

Also I noticed the new file that caused this error was missing the layouts link and so there was no styling and it was missing the header/footer on the page. I have added this also.

museumXTD\Notes\Tableau Acteurs et Projets.md

Missing content below:
---
title: MuseumXTD
layout: layouts/post.njk
tags:
---
